### DISCUSSION TITLE
Next/Image

### DISCUSSION THREAD

**Discussion by karmatradeDev** [nandorojo/solito#85] [Q&A]
Hi wanted to see if anyone has made a hybrid image with dripsy/Image and next/image that when on expo it shows dripsy and when on nextjs it uses next/Image

**Comment by nandorojo** [comment]
could be cool to use react native fast image on native. it’s pretty hard to match the feature parity though, since web has properties like sizes and such.

@axeldelafosse did you do this by chance?

**Reply by axeldelafosse** [reply]
Yep! You can check Universal Image! But it's using Tailwind -- planning to update this soon

**Reply by karmatradeDev** [reply]
Found it! this is awesome thanks a lot you all I'm just starting out with solito and just trying to get a grasp of the methodology which I agree with. Last 2 questions, the '.web.tsx' will it run automagically when using a browser,  And is Component -> index.tsx , index.web.tsx the pattern for cross-platform in solito?

**Reply by nandorojo** [reply]
yes that’s the pattern. you can do the same with `.native.tsx` or `.ios.tsx`, etc. as long as there is a file that ends with .ts/.tsx, you can prefix it with `.<platform>.ts` to specify the same code on that platform.

**Reply by nandorojo** [reply]
i mention this in my talk here, i recommend watching it: https://youtu.be/0lnbdRweJtA

see the “Platform Features” section
